Readme.txt

```diff
@@ -39,144 +40,188 @@ CONFIGURATION
+          is repeated and sometimes with the exchange.
+       3. They may respond with "NR?"
+          You will have to send your exchange again (Hit f2).
+       4. They may respond with "AGN"
+          You will have to send your exchange again (Hit f2).
+       5. "R" with their exchange.
+          When you get the exchange hit enter to log it.
+          Please note that occasionally you will get "NR?" and "AGN" multiple times. Respond with
+          F2 until you get that "R" (Roger) and the exchange.
 
-  PgUp/PgDn, Ctrl-F10/Ctrl-F9, Alt-F10/Alt-F9 - keying speed,
-    in 2 WPM increments. HST Competition uses 5 WPM increments.
-    The default WPM Step rate is 2. Valid values range between 1 and 20.
-    You can override this value by changing the WpmStepRate entry in the
-    MorseRunner.ini file, e.g.:
-            [Settings]
-            WpmStepRate=2
+KEY ASSIGNMENTS
+  F1               - Send CQ (sounds like seek you)
+  F2               - Sends your full exchange
+  F3               - Send TU (thank you)
+  F4               - Sends your call sign
+  F5               - Sends responder's call sign
+  F6               - Send B4
+  F7               - Sends the question mark (?)
+  F8               - Sends NIL (not in log)
+  F11              - Wipes the input fields
+  F12              - sends NR? (request their exchange to be repeated)
+
+  Enter Key        - Enter Sends Message (msg sent depends on state of QSO)
+  Space Bar        - auto completes and jumps between input fields
+  Escape key       - Stops your sending
+
+  Semicolon        - Sends your full exchange
+  Insert key       - Sends your full exchange
+  Plus key (+)     - Send TU (thank you) and saves QSO
+  Period (.)       - Send TU (thank you) and saves QSO
+  Bracket ([)      - Send TU (thank you) and saves QSO
+  Comma (,)        - Send TU (thank you) and saves QSO
+
+  Ctrl-W           - Wipes the input fields
+  Alt-W            - Wipes the input fields (with Windows chime)
+  Shift-Enter      - Saves the QSO without sending anything
+  Ctrl-Enter       - Saves the QSO without sending anything
+  Alt-Enter        - Saves the QSO without sending anything (with Windows Chime)
+  Tab Key          - moves to Call, RST, CQ-Zone, Station Call, and your exchange
+  Shift Tab        - moves to Call, RST, CQ-Zone, Station Call, and your exchange
+  Up arrow         - moves the RIT right (reversible in settings)
+  Down arrow       - moves the RIT left (reversible in settings)
+  Mouse wheel      - moves the RIT
+  Left arrow       - adjusts the RX Bandwidth
+  Right arrow      - adjusts the RX Bandwidth
+  Ctrl-up arrow    - adjusts the RX Bandwidth
+  Ctrl-down arrow  - adjusts the RX Bandwidth
+  Ctrl-mouse wheel - adjusts the RX Bandwidth
+  Page down key    - adjusts the keying speed down
+  Page up key      - adjusts the keying speed up
+  Ctrl-F9          - adjusts the keying speed down
+  Ctrl-F10         - adjusts the keying speed up
```

Comment:
Alt-F9 and Alt-F10 also work to adjust the keying speed

Reply:
I removed these because my Alt-F9 in my windows 10 displays my GeForce Experience program asking me to turn on desktop capture. Alt -F10 does nothing. Is this just my PC or this release? 

(The other two Alt commands gives me the dreaded windows chime - I'd remove them if I could). 

Please check this for me and let me know.. Thanks, Scott

Reply:
Yes, we should add the Alt-F9 and Alt-F10. These choices are still in the code and work on my laptop. Tnx, Mike.
@scotthibbs

Reply:
> I removed these because my Alt-F9 in my windows 10 displays my GeForce Experience program asking me to turn on desktop capture. Alt -F10 does nothing. Is this just my PC or this release?

I would guess your Alt-F9 is mapped for your GeForce program. Thankfully MRCE has many key mappings so you can choose the one(s) that work for you.

Reply:
Add Alt-F9/Alt-F10 to `Readme.txt` file as suggested.